docs/src/operate/c8y/configuration-management.md

```diff
@@ -26,13 +26,18 @@ files = [
   { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf', type = 'c8y-bridge.conf' },
   { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf', type = 'tedge-mosquitto.conf' },
   { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto.conf' },
-  { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o644 }
+  { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o644 },
+  { path = '/etc/containers/certs.d/example/ca.crt', type = 'harbor-certificate', user = 'tedge', group = 'tedge', mode = 0o640, parent_user = 'root', parent_group = 'root', parent_mode = 0o755 },
 ]
 ```
 
 * `path` is the full path to the configuration file.
 * `type` is a unique alias for each file entry which will be used to represent that file in Cumulocity UI.
-* `user`, `group` and `mode` are UNIX file permission settings to be used to create a configuration file. If not provided, the files will be created with `root` user. If the file exists already, its ownership will be retained.
+* `user`, `group` and `mode` are UNIX file permission settings to be used to create a configuration file. If not provided, the files will be created with the current user and group, or `root`.
+  If the file exists already, its ownership will be retained.
```

Comment:
*Preserved* as used bellow seems more appropriate.

```suggestion
  If the file exists already, its ownership will be preserved.
```

Reply:
The text changed in b42bb22fb30b099dfd62868031af710c0cca1055